<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>选项卡1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>分组框</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="4" column="3">
              <widget class="QComboBox" name="comboBox">
               <property name="editable">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>选项一</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>选项二</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>选项三</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>选项四</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>默认按钮</string>
               </property>
               <property name="default">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="radioButton_3">
               <property name="text">
                <string>单选框3</string>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QProgressBar" name="progressBar_3">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit">
               <property name="text">
                <string>read-only</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="3">
              <widget class="QSpinBox" name="spinBox">
               <property name="minimum">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="5" column="4">
              <widget class="QSlider" name="horizontalSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="radioButton">
               <property name="text">
                <string>下拉框可编辑</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QScrollBar" name="horizontalScrollBar">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Flat按钮</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>普通按钮3</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QToolButton" name="toolButton">
               <property name="text">
                <string>工按钮1</string>
               </property>
               <property name="popupMode">
                <enum>QToolButton::MenuButtonPopup</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="5" rowspan="6">
              <widget class="QScrollBar" name="verticalScrollBar">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="text">
                <string>123456</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="radioButton_2">
               <property name="text">
                <string>下拉框不可编辑</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>多选框1</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="checkBox_2">
               <property name="text">
                <string>多选框2</string>
               </property>
              </widget>
             </item>
             <item row="3" column="4">
              <widget class="QProgressBar" name="progressBar_2">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QToolButton" name="toolButton_3">
               <property name="text">
                <string>工具按钮3</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="checkBox_3">
               <property name="text">
                <string>可半选</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QToolButton" name="toolButton_2">
               <property name="text">
                <string>工具按钮2</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QProgressBar" name="progressBar_4">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item row="4" column="4">
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item row="0" column="6" rowspan="6">
              <widget class="QSlider" name="verticalSlider">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="placeholderText">
           <string>这是一段文字</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>选项卡2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QTreeView" name="treeView"/>
          </item>
          <item row="0" column="1">
           <widget class="QListWidget" name="listWidget">
            <item>
             <property name="text">
              <string>项目1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>项目2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>项目3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>项目4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>项目5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>项目6</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QToolBox" name="toolBox">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <widget class="QWidget" name="page">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
             <attribute name="label">
              <string>工具集合1</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="page_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
             <attribute name="label">
              <string>工具集合2</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="page_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
             <attribute name="label">
              <string>工具集合3</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="page_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
             <attribute name="label">
              <string>工具集合4</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_Q"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_C"/>
    <addaction name="action_X"/>
    <addaction name="action_P"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>项目协议知晓情况</string>
     </property>
     <addaction name="action"/>
     <addaction name="separator"/>
     <addaction name="action_4"/>
    </widget>
    <addaction name="action_A_2"/>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>样式选择(&amp;C)</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="actionQDarkStyleSheet_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="separator"/>
   <addaction name="action_S"/>
   <addaction name="action_A"/>
   <addaction name="separator"/>
   <addaction name="action_C"/>
   <addaction name="action_X"/>
   <addaction name="action_P"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="icon">
    <iconset resource="imgRes.qrc">
     <normaloff>:/menu/images/new.png</normaloff>:/menu/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="imgRes.qrc">
     <normaloff>:/menu/images/save.png</normaloff>:/menu/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="imgRes.qrc">
     <normaloff>:/menu/images/saveas.png</normaloff>:/menu/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="icon">
    <iconset resource="imgRes.qrc">
     <normaloff>:/menu/images/exit.png</normaloff>:/menu/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset resource="imgRes.qrc">
     <normaloff>:/menu/images/copy.png</normaloff>:/menu/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="icon">
    <iconset resource="imgRes.qrc">
     <normaloff>:/menu/images/cut.png</normaloff>:/menu/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪贴(&amp;X)</string>
   </property>
  </action>
  <action name="action_P">
   <property name="icon">
    <iconset resource="imgRes.qrc">
     <normaloff>:/menu/images/paste.png</normaloff>:/menu/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>知晓项目协议一</string>
   </property>
  </action>
  <action name="action_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>知晓项目协议二</string>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>默认样式</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>自定义样式</string>
   </property>
  </action>
  <action name="actionQDarkStyleSheet">
   <property name="text">
    <string>QDarkStyleSheet</string>
   </property>
  </action>
  <action name="actiondark">
   <property name="text">
    <string>dark</string>
   </property>
  </action>
  <action name="actionlight">
   <property name="text">
    <string>light</string>
   </property>
  </action>
  <action name="actionQDarkStyleSheet_2">
   <property name="text">
    <string>QDarkStyleSheet</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imgRes.qrc"/>
 </resources>
 <connections/>
</ui>
